metricbeat/module/php_fpm/_meta/config.reference.yml

```diff
@@ -1,5 +1,5 @@
 - module: php_fpm
-  metricsets: ["pool"]
+  metricsets: ["pool", "process"]
```

Comment:
This could be written as a list for reference with non-default metricset commented out, like this:
```
metricsets:
  - pool
  #- process
```

Reply:
hi @jsoriano , I have made the changes in my latest commit.
I have tried using #8292 in order to clean up the handling of the "full" query param (inside the Fetch func) but it seems that I need the "query" config setting to reside somewhere at the metricset level and not at module level as it is at the moment. Am I missing something obvious here?